roles/manifest/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Download Manifest
+  import_tasks: download_manifest.yml
```

Comment:
What's the benefit of having multiple task files each containing one task?

Reply:
I rather like that. What's the downside?

Reply:
More output from Ansible. (As the include is an own task and reported as such)

Reply:
Do you prefer a single files `tasks/main.yml` like:

```yaml
---
- name: Download Subscription Manifest from Red Hat Customer Portal
  theforeman.foreman.redhat_manifest:
    uuid: "{{ manifest_uuid }}"
    username: "{{ rhsm_username }}"
    password: "{{ rhsm_password }}"
    path: "{{ manifest_path }}"
  when: manifest_download

- name: Upload Subscription Manifest to Foreman
  theforeman.foreman.subscription_manifest:
    username: "{{ username }}"
    password: "{{ password }}"
    server_url: "{{ server_url }}"
    organization: "{{ organization }}"
    state: present
    manifest_path: "{{ manifest_path }}"
```

Reply:
Personally? Yes. I wouldn't block the PR on that, though. 

The only reason I ticked "changes required" was the == True part (and the ansible lint run)

Reply:
That's fair, single file makes sense to me!